src/surge-xt/gui/SurgeImageStore.cpp

```diff
@@ -123,6 +125,12 @@ void SurgeImageStore::addEntry(int id)
     SurgeImage *bitmap = new SurgeImage(id);
 
     bitmap_registry[id] = bitmap;
+
+    if (auto *h = SurgeImage::createFromPrefix("hover", id))
+    {
+        std::string name = fmt::format("hover{0}", id);
```

Comment:
Shouldn’t this format be set up to do a 00032

Reply:
Yeah, you're right!